FreeRTOS/FreeRTOS
# Add missing error state assignment

Add missing error state assignment

Description
-----------
QPeek was not setting the error state in one the if statements

Test Steps
-----------


Related Issue
-----------
https://github.com/FreeRTOS/FreeRTOS/issues/136


By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.
